src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
```

Comment:
- [ ] Hello! 😊 As pointed out by the previous reviewer, the initialState should have a state value for books that contains an empty array. 📚 This means that when the initial state is set up, the books property should start with no items in its array, creating a clean and empty slate for your application's book-related data. 📖✨

```sh
// ./src/redux/books/booksSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const booksSlice = createSlice({
  name: "books",
  initialState: {
    books: [],
  },
  reducers: {
    addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
    removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
  },
})

export const { addBook, removeBook } = booksSlice.actions

export default booksSlice.reducer

```